docs/development/accessibility-process.md

```diff
@@ -0,0 +1,191 @@
+    * Maintainers and Developers use the Accessibility Checklist to assess
+      accessibility of their ongoing implementation projects.
+
+
+* [Feature Wiki](https://docu.ilias.de/goto_docu_wiki_wpage_1_1357.html)
+    * For Feature Requests a dedicated section "Accessibility" is to be filled
+      out by the maintainer - very much like "Privacy".
+    * The Maintainer has to state if the feature implements a behaviour that is
+      neither covered by an existing UI-Component nor complies with the accessibility
+      guideline. If this is the case maintainer has to consult with the UI/UX/A11y
+      expert group. The UI/UX/A11y expert group then provides risk analysis of
+      features in the Feature Wiki and recommends an accessible approach. If the
+      maintainer is unsure about the accessibility issues of a feature, the
+      maintainer may turn to Accessibility Mailing List.
+    * The "Accessibility" section is to be completed before the article can be
+      finally decided by Jour Fixe.
+
+* [Testrail](https://testrail.ilias.de)
+    * A [test suite dedicated to Accessibility](https://testrail.ilias.de/index.php?/runs/view/566&group_by=cases:section_id&group_order=asc)
+      is available.
+    * Test cases are well groomed, easy to carry out and understand.
+    * Test cases refer to resource on specific accessibility issues for all
+      consumers to look up and self-educate
+    * Test Cases are provided that use simple diagnostical browser extensions like
+      lighthouse, HTML Validator or Wave browser extension.
+    * All KS-Entries are tested for the prioritized issues and using automatic
+      tools.
+
+* Development Tools - suggested WAVE API
+    * Detects issues very early on and drives home the importance of accessibility
+      during development.
+    * Reports are prepared with Continuous Integration and reported to Jour Fixe.
+    * All developers are encouraged to make use of simple diagnostical browser
+      extensions like lighthouse, HTML Validator or Wave browser extension
+
+* [Issue Tracker](https://mantis.ilias.de)
+    * Issues can be reported on automatically testable violations of guidelines.
+    * Issues can be reported on violations of accessibility rules listed in a KS
+      entry, but also the on the rules themselves if they need improvement.
+      The KS entry and implementation are amended via the normal KS process.
+    * Issues can be reported against soft / non-objective parts of the accessibility
+      guidelines. These require discussion and decisions. Accessibility Guidelines
+      are to be amended or clarified to become more objective. Tickets of this kind
+      are assigned to the UI/UX/A11y expert group.
+
+* [Jour Fixe](https://docu.ilias.de/goto.php?target=wiki_1357_Jour_Fixe_Agendas)
+    * Maintainers can put forward their Accessibility issues that came up during
+      implementation as "Development Issues".
+
+* Accessibility Mailing List
```

Comment:
Brauchen wir das in Zukunft in dieser Form? Wollen wir eine zentrale Anlaufstelle für die UI/UX/A11y-Gruppe vorgeben?

Reply:
-1 für Mailing list. Wir sollten da auf die Expert Group verweisen mit TB als Fallback solange keine besteht. Ja, zentrale Anlaufstelle wäre gut, aber z.Z. noch nicht vorhanden.

Reply:
+1 Timon, nicht noch eine Mailing-List. Ich würde mir hier auch andere Kanäle vorstellen (Tracker, direkte E-Mails an Experten, ein Bereich auf der docu-ILIAS, o.ä,).